syntax = "proto3";


service Auth {
    rpc Authenticate (AuthenticateRequest) returns (AuthenticateResponse);
}

message AuthenticateRequest {
    string login = 1;
    string password = 2;
}

message AuthenticateResponse {
    bool allowed_access = 1;
}
